pkgs/applications/video/screenkey/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, substituteAll
+, buildPythonApplication
+, fetchFromGitHub
+, distutils_extra
+, setuptools-git
+, intltool
+, pygtk
+, libX11
+, libXtst
+, wrapGAppsHook
+, defaultIconTheme
+, hicolor_icon_theme
+}:
+buildPythonApplication rec {
+  pname = "screenkey";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "wavexx";
+    repo = "screenkey";
+    rev = "screenkey-${version}";
+    sha256 = "14g7fiv9n7m03djwz1pp5034pffi87ssvss9bc1q8vq0ksn23vrw";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./paths.patch;
+      inherit libX11 libXtst;
+    })
+  ];
+
+  nativeBuildInputs = [
+    distutils_extra
+    setuptools-git
+    intltool
+
+    wrapGAppsHook
+  ];
+
+  propagatedBuildInputs = [
+    pygtk
+
+    defaultIconTheme
```

Comment:
The icon themes do not need to be propagated, actually the `hicolor_icon_theme` is only used for setup hook.

Reply:
Oops. Wrong section.

Reply:
Fixed.